package-lock.json

```diff
@@ -20643,6 +22289,24 @@
         "node": "^16.20.0 || ^18.0.0 || ^20.0.0 || ^22.0.0 || ^24.0.0"
       }
     },
+    "packages/react-native-lockdown": {
+      "name": "@lavamoat/react-native-lockdown",
+      "version": "0.0.1-beta.0",
+      "license": "MIT",
+      "dependencies": {
+        "ses": "^1.12.0"
+      },
```

Comment:
since 1.13.0 isn't released yet

it can be built in endo repo and used via patch-package in apps
(or yarn packed then added as a dep or resolution)

Reply:
So we need `ses@1.13.0` for this PR to pass CI, right?